indico/modules/categories/operations.py

```diff
@@ -20,6 +20,24 @@
 
 from indico.core.db import db
 from indico.modules.categories import logger
+from indico.modules.categories.models.categories import Category
+
+
+def create_category(parent, data):
+    category = Category(parent=parent)
+    data.setdefault('default_event_themes', parent.default_event_themes)
+    data.setdefault('timezone', parent.timezone)
```

Comment:
Is `setdefault` particularly useful here? Why not `data.update({'default_event_themes': ...})`?